### DISCUSSION TITLE
Display as MudSwitch for boolean type to table instead of text

### DISCUSSION THREAD

**Discussion by kirkey** [fullstackhero/blazor-wasm-boilerplate#211] [Ideas]
Would like to know if it's possible to show it as MudSwitch for bolean type or other MudBlazor components to table.
`protected override void OnInitialized() =>
        Context = new(
            entityName: "Adjustment",
            entityNamePlural: "Adjustments",
            entityResource: FSHResource.Adjustments,
            fields: new()
            {
                new(data => data.AdjustmentType, "Type", "AdjustmentType"),
                new(data => data.Number, "Number", "Number"),
                new(data => data.Name, "Name", "Name"),
                new(data => data.Amount.ToString("N2"), "Amount", "Amount"),
                new(data => data.IsMonthMid, "Mid", "IsMonthMid"),
                new(data => data.IsMonthEnd, "End", "IsMonthEnd"),
                new(data => data.Description, "Description", "Description"),
                new(data => data.Notes, "Notes", "Notes"),
            },
            idFunc: Adjustment => Adjustment.Id,
            searchFunc: async filter => (await Client
                .SearchAsync(filter.Adapt<AdjustmentSearchRequest>()))
                .Adapt<PaginationResponse<AdjustmentDto>>(),
            createFunc: async Adjustment => await Client.CreateAsync(Adjustment.Adapt<AdjustmentCreateRequest>()),
            updateFunc: async (id, Adjustment) => await Client.UpdateAsync(id, Adjustment),
            deleteFunc: async id => await Client.DeleteAsync(id),
            exportAction: string.Empty);`

If someone already did, please show us how. Thank you!

**Comment by fretje** [comment]
There is already a way to show a checkbox if you set the type to "bool":

https://github.com/fullstackhero/blazor-wasm-boilerplate/blob/698feea2f20c51c7d7b13629815018204a14a941/src/Client/Components/EntityTable/EntityTable.razor#L103-L106

I think the easiest would be to change it there to show a mudswitch instead... keep in mind this will then change for all the fields with type "bool"...
